doc/user-guides/gateway-rl-for-cluster-operators.md

```diff
@@ -1,244 +1,204 @@
+                                 └─────────┬────────┘
+                                           │
+                                 ┌─────────┴────────┐
+                                 │   (HTTPRoute)    │
+                                 │     toystore     │
+                                 │                  │
+                                 │ *.toystore.io    │
+                                 │ *.toystore.local │
+                                 └────────┬─────────┘
+                                          │
+                                   ┌──────┴───────┐
+                                   │   (Service)  │
+                                   │   toystore   │
+                                   └──────────────┘
 ```
 
-### Rate limiting Gateway traffic
-
-![](https://i.imgur.com/0o3yQzP.png)
+Deploy the sample API:
 
-RateLimitPolicy applied for the Gateway.
+```sh
+kubectl apply -f examples/toystore/toystore.yaml
+```
 
-| Policy        |                         Rate Limits |
-|---------------|------------------------------------:|
-| `POST /*`     |  **2** reqs / **10** secs (0.2 rps) |
-| Per remote IP | **25** reqs / **10** secs (2.5 rps) |
+Route traffic to the API from both gateways:
 
 ```sh
 kubectl apply -f - <<EOF
----
-apiVersion: kuadrant.io/v1beta1
-kind: RateLimitPolicy
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
 metadata:
-  name: kuadrant-gw
-  namespace: istio-system
+  name: toystore
 spec:
-  targetRef:
-    group: gateway.networking.k8s.io
-    kind: Gateway
-    name: istio-ingressgateway
-  rateLimits:
-    - rules:
-      - methods: ["POST"]
-      configurations:
-        - actions:
-            - generic_key:
-                descriptor_key: expensive_op
-                descriptor_value: "1"
-      limits:
-        - conditions: ["expensive_op == '1'"]
-          maxValue: 2
-          seconds: 10
-          variables: []
-    - configurations:
-        - actions:
-            - remote_address: {}
-      limits:
-        - conditions: []
-          maxValue: 25
-          seconds: 10
-          variables: ["remote_address"]
+  parentRefs:
+  - name: external
+    namespace: istio-system
+  - name: internal
+    namespace: istio-system
+  hostnames:
+  - "*.toystore.io"
+  - "*.toystore.local"
+  rules:
+  - backendRefs:
+    - name: toystore
+      port: 80
 EOF
 ```
 
-### Validating the rate limit policies (HTTPRoute and Gateway).
+### ⑤ Verify the rate limiting works by sending requests in a loop
+
+Expose the gateways, respectively at the port numbers `9080` and `9081` of the local host:
 
-`GET /toy` @ **1** rps (expected to be rate limited @ **8** reqs / **10** secs (0.8 rps))
 ```sh
-while :; do curl --write-out '%{http_code}' --silent --output /dev/null -H "Host: api.toystore.com" http://localhost:9080/toy | egrep --color "\b(429)\b|$"; sleep 1; done
+kubectl port-forward -n istio-system service/external-istio 9080:80 2>&1 >/dev/null &
```

Comment:
using [Kind config port mapping here](https://github.com/Kuadrant/kuadrant-operator/blob/main/utils/kind-cluster.yaml#L10), we are already mapping port `9080` to the istio's default ingress controller `istio-ingressgateway`. This is implemented using iptables chains or whatever (actually I don't know). The thing is that when you run `kubectl port-forward` local port 9080, it does not return the typical error `port is in use...`. Instead the traffic in local port 9080 is still going to the `istio-ingressgateway`. Local iptable chains win over kubectl port-forward :shrug: 

It can be fixed mapping local port 9081 to `external` and local port `9082` to internal

Reply:
Done as suggested.